src/sentry/api/endpoints/user_details.py

```diff
@@ -89,3 +97,69 @@ def put(self, request, user):
             return Response(serialize(user, request.user))
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    @sudo_required
+    def delete(self, request, user):
+        """
+        Delete User Account
+
+        Also removes organizations if they are an owner
+        :pparam string user_id: user id
+        :param list organizations: List of organization ids to remove
+        :auth required:
+        """
+
+        # from `frontend/remove_account.py`
+        org_list = Organization.objects.filter(
+            member_set__role=roles.get_top_dog().id,
+            member_set__user=user,
+            status=OrganizationStatus.VISIBLE,
+        )
+        org_results = []
+        for org in sorted(org_list, key=lambda x: x.name):
+            # O(N) query
+            org_results.append({
+                'organization': org,
+                'single_owner': org.has_single_owner(),
+            })
+
+        avail_org_slugs = set([o['organization'].slug for o in org_results])
+        orgs_to_remove = set(request.DATA.get('organizations')).intersection(avail_org_slugs)
```

Comment:
you should do `set(request.DATA.get('organizations', []))` because `set(None)` raises a `TypeError`

and actually, you should probably do at least some minimal validation that if the key does exist, it's a list